docs/_layouts/global.html

```diff
@@ -17,113 +19,133 @@
-                            <div class="dropdown-menu" aria-labelledby="navbarDeploying">
-                                <a class="dropdown-item" href="cluster-overview.html">Overview</a>
-                                <a class="dropdown-item" href="submitting-applications.html">Submitting Applications</a>
-                                <div class="dropdown-divider"></div>
-                                <a class="dropdown-item" href="spark-standalone.html">Spark Standalone</a>
-                                <a class="dropdown-item" href="running-on-mesos.html">Mesos</a>
-                                <a class="dropdown-item" href="running-on-yarn.html">YARN</a>
-                                <a class="dropdown-item" href="running-on-kubernetes.html">Kubernetes</a>
-                            </div>
-                        </li>
-
-                        <li class="nav-item dropdown">
-                            <a href="#" class="nav-link dropdown-toggle" id="navbarMore" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">More</a>
-                            <div class="dropdown-menu" aria-labelledby="navbarMore">
-                                <a class="dropdown-item" href="configuration.html">Configuration</a>
-                                <a class="dropdown-item" href="monitoring.html">Monitoring</a>
-                                <a class="dropdown-item" href="tuning.html">Tuning Guide</a>
-                                <a class="dropdown-item" href="job-scheduling.html">Job Scheduling</a>
-                                <a class="dropdown-item" href="security.html">Security</a>
-                                <a class="dropdown-item" href="hardware-provisioning.html">Hardware Provisioning</a>
-                                <a class="dropdown-item" href="migration-guide.html">Migration Guide</a>
-                                <div class="dropdown-divider"></div>
-                                <a class="dropdown-item" href="building-spark.html">Building Spark</a>
-                                <a class="dropdown-item" href="https://spark.apache.org/contributing.html">Contributing to Spark</a>
-                                <a class="dropdown-item" href="https://spark.apache.org/third-party-projects.html">Third Party Projects</a>
-                            </div>
-                        </li>
-
-                        <li class="nav-item">
-                            <input type="text" id="docsearch-input" placeholder="Search the docs…">
-                        </li>
-                    </ul>
-                    <!--<span class="navbar-text navbar-right"><span class="version-text">v{{site.SPARK_VERSION_SHORT}}</span></span>-->
+                <div class="row mt-5">
+                  <div class="col-12 col-lg-6 no-gutters">
+                    Apache Spark is a unified analytics engine for large-scale data processing.
```

Comment:
Nit: how about having a headline as the Spark website does
```
<h2>What is Apache Spark<span class="tm" style="bottom: 14px;">&trade;</span>?</h2>
```

Reply:
I changed it like this, the problem is that the default headers on the page are h1, if we make the what is spark h2, it looks a bit odd.

h2 
<img width="1057" alt="SCR-20230319-e8x" src="https://user-images.githubusercontent.com/3421/226165428-13cedb4f-2344-41b2-84a2-72fd5a70abe7.png">

h1

As an h1 it looks even more odd

<img width="1058" alt="SCR-20230319-e9z" src="https://user-images.githubusercontent.com/3421/226165475-70b85a9f-20f4-4f2e-b523-d73458e1ffa5.png">


I don't have a preference between the current version in the PR and the h2 version. Please let me know.

Reply:
Since we are not changing the title (https://github.com/apache/spark/pull/40269#discussion_r1140751129), let's keep this one as well.